.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: ✨ Feature Request
+about: Suggest an idea for this project
+title: "[REQUEST] "
+labels: feature request
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Why bold for headings here when the other template uses real headings?

Reply:
This one was copied directly from GitHub with only minor changes.

I based the bug template on React's:
https://github.com/facebook/react/blob/main/.github/ISSUE_TEMPLATE/bug_report.md